parlai/agents/fairseq/fairseq.py

```diff
@@ -447,6 +447,9 @@ def save(self, path):
                 del self.opt['override']
             json.dump(self.opt, handle)
 
+        # force save the dict
+        self.dict.save(path + '.dict', sort=True)
```

Comment:
I don't think we sort=True here, because that is destructive on the ordering. At this point, we absolutely want the ordering to stay fixed.